README.md

```diff
@@ -106,27 +107,73 @@ _This is also how you would use this module without `@decorator` syntax, althoug
 
 ### Usage with React Hooks
 
-Following the example above, once at least one component is wrapped with `@track()` we can access a `tracking` object via the `useTracking` hook from anywhere in the sub-tree:
+Following the example above, we can also access the `trackEvent` method via the `useTracking` hook from anywhere in the tree:
 
 ```js
 import { useTracking } from 'react-tracking';
 
-const SomeChild = () => {
-  const tracking = useTracking();
+const FooPage = () => {
+  const { trackEvent } = useTracking({ page: 'FooPage' });
 
   return (
     <div
       onClick={() => {
-        tracking.trackEvent({ action: 'click' });
+        trackEvent({ action: 'click' });
       }}
     />
   );
 };
 ```
 
-The `useTracking` hook returns an object with the same `getTrackingData()` and `trackEvent()` methods that's provided as `props.tracking` when wrapping with the `@track()` decorator/HoC.
+The `useTracking` hook returns an object with the same `getTrackingData()` and `trackEvent()` methods that are provided as `props.tracking` when wrapping with the `@track()` decorator/HoC. It also returns an additional property on that object: a `Track` component that can be returned as the root of your component's sub-tree to pass any new contextual data to its children.
+
+> Note that if you need contextual tracking data to be passed to child components, you need to wrap the markup returned by your component with `Track`. This will create a new context with the new tracking data merged in, and pass it to all child components:
```

Comment:
```suggestion
> Note that in most cases you would wrap the markup returned by your component with `<Track />`. This will merge new tracking context and make it available to all child components. The only time you _wouldn't_ wrap your returned markup with `<Track />` is if you're on some leaf component and don't have any more child components that need tracking info.```